src/pages/index.html

```diff
@@ -4,16 +4,380 @@
+    </label>
+  </div>
+
+  </div>
+      <div class="visa">
+        <i class="  fa-brands fa-cc-visa"></i> 
+        <i class="fa-brands fa-cc-mastercard"></i>
+                  <i class=" fa-brands fa-cc-visa">
+          </i><i class="fa-brands fa-cc-visa"></i>   
+          
+          
+        </div>
+      </div>
+    </div>
+
+  </div>
+  </div>
+  <div class="container">
+
+<div class="Resource-Management">
+  <div class="Management ">
+    <div class="box">
+      <img src="../assets/images/Layer 1.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 2.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 3.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 4.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer5.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 6.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div>
+  </div>
+
+</div>
+</div>
+
+<div class="customers ">
+
+  <div class="make">
+    <h3> See what our customers are saying  </h3>
+    <p> 2,157 people have said how good we are  </p>
+  </div>
+   
+
+<div class="people ">
+
+
+
+
+<div class="card1" >
+
+<img src="../assets/images/BG (1).png" alt="">
+    <p >“I can’t tell how easy it was to grow my small shop with it”</p>
+    <h5>Jenny Wilson</h5>
+<span> Founder, ExtendSales </span>
+  </div>
+
+  <div class="card1" >
+
+
+    <img src="../assets/images/BG.png" alt="">
+    <p >“I can’t tell how easy it was to grow my small shop with it”</p>
+      <h5>Jenny Wilson</h5>
+  <span> Founder, ExtendSales </span>
+    </div>
+
+</div>
+</div>
+<div class="decision ">
+  <div class="make">
+    <h3>Make the wise decision for your business  </h3>
+    <p> Choose from our affordable 3 packages  </p>
+  </div>
```

Comment:
![Capture](https://github.com/shaban011147/landing/assets/106445705/846448cf-8f50-4dc1-bf45-26328a53626a)
 
margin bottom in h3 not equal 8px but equal 20px

Reply:
done